### PR TITLE
small changes and cleanup

### PR DESCRIPTION
**Pull request by MilanHalada**

### CODE REVIEW

File: LunchTime/wwwroot/js/lunchtime.js [hunk 1]
```diff
@@ -59,4 +59,22 @@ function cityChanged() {
...
+        type: 'GET',
+        url: '/Home/Menus',
+        success: (html) => {
+            const isLoaded = xhr.getResponseHeader('isLoaded');
+            console.log('headers', xhr.getAllResponseHeaders());
+            console.log('isLoaded header', isLoaded);
+            if (isLoaded === 'True' || isLoaded === 'true' ) {
+                $('#main').html(html);
+            } else {
+                setTimeout(() => loadData(), 200);
```
Comment (oskrabanek) [inline LunchTime/wwwroot/js/lunchtime.js]: Possible unhandled StackOverflow exception

File: LunchTime/Managers/LunchProvider.cs [hunk 2]
```diff
@@ -3,92 +3,49 @@
...
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Mvc.Rendering;
 using Newtonsoft.Json;
 
 namespace LunchTime.Managers
 {
     public class LunchProvider : ILunchProvider
     {
-        private readonly IHttpContextAccessor _httpContextAccessor;
+        private static readonly ILog _log = LogManager.GetLogger(typeof(MenusProvider));
```
Comment (oskrabanek) [inline LunchTime/Managers/LunchProvider.cs]: Generaly, it would be better to have the logger injected (Mocking for unit testing)

**Review by oskrabanek — COMMENTED** [review]
The changes look OK, thank you for your contribution.

File: LunchTime/wwwroot/js/lunchtime.js (hunk 1 above)
Comment (MilanHalada) [reply LunchTime/wwwroot/js/lunchtime.js]: Fixed with next commit

File: LunchTime/Managers/LunchProvider.cs (hunk 2 above)
Comment (MilanHalada) [reply LunchTime/Managers/LunchProvider.cs]: changed in next commit

**Pull request closed without merging**